deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/README.md

```diff
@@ -0,0 +1,114 @@
+# Dashboard Saturation Tests
+
+This module contains a basic set of tests for the Wazuh Dashboard. The tests are performed using Artillery and Playwright. The tests are executed through a Python script.
+
+The Python script needs some input parameters to perform the tests and get the results. Artillery gets the test results in JSON format and the script converts them to CSV format so they can be analyzed more easily.
+
+```shell script
+dashboard_saturation_tests/
+├── data/
+│   ├── lib/
+│   │   ├── CookieManager.js
+│   │   ├── PathManager.js
+│   │   └── ScreenshotManager.js
+│   ├── tests/
+│   │   ├── EndpointTest.js
+│   │   ├── EventTest.js
+│   │   ├── LoginTest.js
+│   │   └── OverviewTest.js
+│   ├── __init__.py
+│   ├── artillery.xml
+│   ├── dashboard_saturation_tests.py
+│   └── processor.js
+├── README.md
+├── requirements.txt
+├── setup.py
+└── version.json
```

Comment:
It is necessary to update the directory structure as some have changed or are no longer there (`requirements.txt`, `setup.py`, etc.).